SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -247,6 +247,9 @@
   @JsonProperty
   private String shellCommandUserPlaceholder = "{USER}";
 
+  @JsonProperty
+  private String shellCommandPidFile = ".shell_command_pid";
```

Comment:
this is the PID file for the command that the shell command should operate on, right? `.shell_command_pid` seems a little ambiguous to me, can we make it clearer? maybe `.task-pid`?